src/RequestParserFactory.php

```diff
@@ -13,4 +13,9 @@ public function createQueryParser();
      * @return RequestParser
      */
     public function createBodyParser();
+
+    /**
+     * @return RequestParser
+     */
+    public function createCookieParser();
```

Comment:
This might break existing classes implementing the interface. What do you think about creating a second interface `CookieParserFactory` with just the `createCookieParser`? Then the specific `*RequestParserFactory` implementations can just `implements RequestParserFactory, CookieParserFactory`

Reply:
Yep, true 👍 will change that